nixos/modules/services/x11/xbanish.nix

```diff
@@ -0,0 +1,35 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let cfg = config.services.xbanish;
+
+in {
+  options.services.xbanish = {
+
+    enable = mkOption {
+      type = types.bool;
+      default = false;
+      example = true;
+      description = "Enable xbanish to hide your mouse pointer while not in use";
+    };
+
+    arguments = mkOption {
+      description = "Arguments to pass to xbanish command";
+      default = "";
+      example = "-d -i shift";
+      type = types.str;
+    };
+  };
+
+  config = mkIf cfg.enable {
+    systemd.user.services.xbanish = {
+      description = "xbanish";
+      wantedBy = [ "default.target" ];
```

Comment:
updated the description in the service and the package. 

i'm basing the service off of unclutter and redshift, which were both recently updated to use user services (e34814e3173bb112aed3be671a05d699285d55dc, 882391a1620f5179eab0f9f4b96f85af875cbb34 respectively). @rnhmjoj @fpletz why are these using `default.target` instead of `graphical.target`?